regression/check_submodules.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
yeah I agree, but would renaming this break other things? There are a bunch of existing makefiles have have the word regression in them.

Reply:
Right, let me slightly modify my request, then. Since this specific check isn't referenced by the existing Makefile in this `regression` directory, how about we just move this one script into a new directory?

Reply:
ok